src/client/components/Forms/SignUp.js

```diff
@@ -1,22 +1,90 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import iconGithub from '../../assets/images/icons/github.svg';
+import './Signup.style.css';
+import Button from '../Button/Button';
 
 export default function SignUp({ onSubmit }) {
   const [email, setEmail] = useState('');
+  const [fullName, setFullName] = useState('');
+  const [position, setPosition] = useState('');
+  const [profileImageUrl, setProfileImageUrl] = useState('');
+  const [linkedin, setLinkedin] = useState('');
+  const [github, setGithub] = useState('');
+  const [website, setWebsite] = useState('');
   const [password, setPassword] = useState('');
   const [passwordConfirm, setPasswordConfirm] = useState('');
+  const [validation, setValidation] = useState('');
+
   const handleEmailInput = (e) => setEmail(e.target.value);
   const handlePasswordInput = (e) => setPassword(e.target.value);
   const handlePasswordConfirmInput = (e) => setPasswordConfirm(e.target.value);
+  const handleFullNameInput = (e) => setFullName(e.target.value);
+  const handlePositionInput = (e) => setPosition(e.target.value);
+  const handleProfileImageUrlInput = (e) => setProfileImageUrl(e.target.value);
+  const handleLinkedinInput = (e) => setLinkedin(e.target.value);
+  const handleGithubInput = (e) => setGithub(e.target.value);
+  const handleWebsiteInput = (e) => setWebsite(e.target.value);
   const handleSubmit = (e) => {
     e.preventDefault();
-    onSubmit({ email, password, passwordConfirm, fullName: 'Jane Doe' });
+    if (password !== passwordConfirm) {
+      return setValidation('**Your password does not match**');
+    }
+    setValidation('');
+    onSubmit({
+      email,
+      password,
+      passwordConfirm,
+      fullName: 'Jane Doe',
```

Comment:
Don't hard code any values, why `fullName: 'Jane Doe'` ?

Reply:
yes, you dont need to hard code this as a default value. We had a call where i communicated that to you @kamalrsa ? That is where I suggested the other props to match my part of the functionality like website and profileImageUrl. We also changed your name to fullName. So im not sure why now you left it hard coded? We had several conversations about this Kamal. I´m surprised its still not clear to you..

Reply:
I know you used for testing but this is alreday merged in my component.